.github/workflows/lint.yml

```
@@ -19,5 +19,5 @@ jobs:
 
       - name: Run lint
         run: |
-          yarn install --frozen-lockfile
+          yarn install
```

Seems like this was pushed as a fix for linting, which does not seem right? is there any other reason for this or can we revert?

```suggestion
          yarn install --frozen-lockfile
```

---

Let's revert it. Thanks for checking this out.

---

resolved on the latest commit